man-roxygen/arg_observer.R

```diff
@@ -0,0 +1,2 @@
+#' @param observer [\code{character(1)}]\cr
```

Comment:
and what the Oberserver does should be doced in initCocoObserver, which is linked anyhow when the class is correctly mentioned here...